python/dgl/backend/mxnet/tensor.py

```diff
@@ -17,7 +18,10 @@
 
 # After MXNet 1.5, empty tensors aren't supprted by default.
 # After we turn on the numpy compatible flag, MXNet supports empty NDArray.
-mx.set_np_shape(True)
+dgl_mx_np_shape = os.environ.get('DGL_MXNET_NP_SHAPE', '1')
+if dgl_mx_np_shape.isdigit():
+    if int(dgl_mx_np_shape) == 1:
+        mx.set_np_shape(True)
```

Comment:
I think it's better just throw error if the env var is not of the required type. How about just:
```python
mx.set_np_shape(bool(os.environ.get('DGL_MXNET_SET_NP_SHAPE', True)))
```

Reply:
Sounds easier